### ISSUE TITLE
Check failed: (size_t)batchSize == idLabel->getSize() (1 vs. 32)

### ISSUE THREAD

**Issue by shibing624** [PaddlePaddle/Paddle#6329]
出core:Check failed: (size_t)batchSize == idLabel->getSize() (1 vs. 32)

batch_size 设为1时，可以运行通过。

请问 怎么计算语言模型困惑度的值，sum_cost 计算的时候，我是这样写的：.

left_output = paddle.layer.fc(input=[features[0]], size=vocab_sizes[0], act=paddle.activation.Softmax())

left_entropy = paddle.layer.cross_entropy_cost(input=left_output, label=left_target)
left_entropy_p = paddle.evaluator.value_printer(left_entropy)
left_trans = paddle.layer.trans(left_entropy)
left_score = paddle.layer.sum_cost(left_trans)

这样可以计算一个句子的困惑度值，但没法处理一个batch的。

**Comment by jacquesqiao** [comment]
left_target 这个label的size是不是不对？

**Comment by shibing624** [comment]
left_target这个label的size是对的，是一个sequence的。：left_target = paddle.layer.data(name="left_target", type=paddle.data_type.integer_value_sequence(vocab_sizes[0]))
